### PR TITLE
Navbar register option

### LINKED ISSUE #105: Register option in NavBar menu
# Expected Behavior

There should be a register option under login in navbar menu

# Current Behavior

Right now register can only be accessed by manually going to the route
screenshot_20190304_144346

## Steps to Reproduce

A similar menu item such as login which point to /register needs to be added

### PR DESCRIPTION
**Pull request by sweezy793**
Earlier register route can only be accessed by manually entering that route. Now user can register directly through register button in navbar

**Closing issues**

Closes #105

### CODE REVIEW

**Comment by sweezy793** [comment]
@ivantha Please review the changes :)

**Comment by sweezy793** [comment]
> Why keys.js.example is deleted

Sorry, actually I renamed keys.example.js to keys.js. I will just revert it back.

File: fact-bounty-server/config/keys.example.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 module.exports = {
 	mongoURI: 'MONGO_URL',
 	secretOrKey: 'secret'
-}
+}
```
Comment (Anmolbansal1) [inline fact-bounty-server/config/keys.example.js]: Insert a new line at end of file

Comment (sweezy793) [reply fact-bounty-server/config/keys.example.js]: Done!

**Comment by sweezy793** [comment]
> Otherwise looks good

@Anmolbansal1 Please approve the changes :)

**Comment by root5533** [comment]
This issue solved in #109

**Pull request closed without merging**